vignettes/including-adam-data-in-teal.Rmd

```diff
@@ -1,33 +1,27 @@
 ---
-title: "Input `ADaM` data in a teal application"
+title: "Using `ADaM` Data in `teal` Applications"
```

Comment:
@chlebowa I think ` in titles are not recognized by pkgdown. Can you revert backticks in titles on pkg names?
Check this out
https://insightsengineering.github.io/teal.transform/latest-tag/articles/teal-transform.html
![image](https://github.com/insightsengineering/teal/assets/133694481/03281e12-8432-4298-9937-b166b1526711)